src/main/java/PlayRace.java

```diff
@@ -0,0 +1,36 @@
+public class PlayRace {
+    private Car cars[];
```

Comment:
C 스타일의 배열 선언이네요. 자바는 객체 지향 언어입니다. 객체를 활용하면 어떨까요?
데이터를 배열로 구성하지 말고, 컬렉션으로 구성하면 어떨까요? 

```suggestion
    private List<Car> cars = new ArrayList<>();
```